Milestone3/Frontend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
Change /node_modules to node_modules without the slash. (You should add a gitignore for milestone 4 as well. )